admin-docs/creating-entity-pages.md

```diff
@@ -0,0 +1,100 @@
+```
+
+This already wraps the entity's title within a paragraph tag element.
+
+Accessing the entity's metadata requires a little more knowledge into the way Uwazi stores the data, but here is a brief explanation of what you can expect. For the following entity example:
+
+| Properties       | Values                                           | Observations                                                   |
+| ---------------- | ------------------------------------------------ | -------------------------------------------------------------- |
+| Title            | Title of Entity                                  |                                                                |
+| Country          | India                                            | Data based on a Thesaurus of countries                         |
+| Description      | A rich text description                          |                                                                |
+| Related Entities | Title of another entity, Title of a third entity | Data based on two relationships selected in Relationship field |
+
+You need to know that every entity has its `metadata` stored in `entity.metadata`. This is an object where property names are the keys (sanitized to lower case and spaces replaced with underscores) and every value is actually an array of objects that have either `value` or `label` or both keys, depending on their type. So, internally, the above entity will look like:
+
+```
+{
+  _id: "89af7g8ad98ads"
+  title: "Title of Entity"
+  metadata: {
+    country: [{ label: "India", key: "df9a88" }],
+    description: [{ value: "A rich text description" }],
+    related_entities: [
+      { label: "Title of another entitiy", value: "9ad8f212" },
+      { label: "Title of a third entity", value: "as182736a" },
+    ]
+  }
+}
+```
+
+So, to expand on the previous explanation, the way to extract the data for properties other than `title`, is with something like this:
+
+```
+<Value path="currentEntity.metadata.country.0.value />
+```
+
+This will print the value `India` on the page's HTML.
+
+As noted, you could use the `Repeat` component to list all the "Related Entities". Here is a more in depth example of how to create a more complex display of property name and property values:
+
+```
+<p><Value path="currentTemplate.properties.2.label" />: </p>
+<ul>
+  <Repeat path="currentEntity.metadata.related_entities">
+    <li>
+      <Value path="label" />
+    </li>
+  </Repeat>
+</ul>
+```
+
+This will print:
+
+---
+
+Related Entities:
+
+- Title of another entitiy
+- Title of a third entity
+
+---
+
+Notice that the position of the property in `currentTemplate` (in this case "2") needs to be inspected via the API or analyzing the respose to the request on the browser. There is no way to know before hand the position of a property in the template. In the entites, they are namespaced by property name, so it is easier to extract the data.
```

Comment:
Well, this already does that.  It queries the API and brings the currentTemplate.  The problem is that properties are stored as arrays, so you need to fetch `property[2]` instead of doing it by key.  But I get the wording error on my part, I will try to correct it.